x-pack/plugins/security_solution/server/lib/detection_engine/rule_management/api/rules/bulk_actions/route.ts

```diff
@@ -128,12 +141,34 @@ const buildBulkResponse = (
     deleted?: RuleAlertType[];
   }
 ) => {
+  // Split errors into expected skipped rules and other errors
+  const { skipped, errors: errorsWithoutSkipped } = errors.reduce<ErrorsAndSkippedRules>(
+    (acc: ErrorsAndSkippedRules, error: BulkActionError) => {
+      if (isBulkEditError(error)) {
```

Comment:
It better to replace double `if` with one

`if (isBulkEditError(error) && Object.values(BulkEditSkipReason).includes(error.message as BulkEditSkipReason)) {`

Reply:
Adressed 👍